### ISSUE TITLE
Virtual document for screen reader announces document three times.

### ISSUE THREAD

**Issue by smartguest** [microsoft/vscode#141903]
- VS Code Version: N/A
- OS Version: Windows 10

Using screen reader NVDA.

Steps to Reproduce:

1. Open a notebook in vscode, with NVDA active.
2. Tab focus on the first few focusable items in the notebook, such as a code button or add code cell,
3. Shift tab until it focuses on the document window as a whole.
4. NVDA will announce "Virtual Document, Document, Document" confusingly when this is done.

The screenshot below shows the transcript of the text when this happens:
![Virtual Document Screenshot](https://user-images.githubusercontent.com/18018452/151870280-4dc70c8e-19f9-44f2-86f4-c22170a29402.png)

This was first reported in here: occurring with an HTML webview window for the vscode-mssql extension

https://github.com/microsoft/vscode-mssql/issues/17197

**Comment by roblourens** [comment]
I assume that it is coming from the two levels of iframes. "Virtual Document" is the title of the webview, do we need that title @mjbvz? 

And the `document` might be read because they have `role=document`, is that attribute required?

**Comment by roblourens** [comment]
I see https://github.com/microsoft/vscode/pull/93435. Do we need it on both or is it possible to have it on just one iframe?

**Comment by mjbvz** [comment]
@roblourens I honestly don't know. Only the inner most iframe is technically the "document" but someone who knows more about how screen readers work would need to chime in

**Comment by roblourens** [comment]
It should be safe to remove the title right? Just fixing that removes a lot of noise, I'll do that

**Comment by roblourens** [comment]
Yes, I removed the title. I think that's the only thing I wanted to do in this issue so I will close it.

**Comment by smartguest** [comment]
Pleaes reopen this issue as it appears to have only been partially fixed for our primary issue. 

While "virtual document" is no longer announced, the way vscode seemingly handles html content appears to be causing issues with the screen reader reading multiple "document" tags. 

See comment here for more details:
https://github.com/microsoft/vscode-mssql/issues/17197#issuecomment-1158288492

**Comment by roblourens** [comment]
We can see if `role=document` makes a difference but it might just read that whenever you enter an iframe, in that case we might not be able to fix it

**Comment by Roopesh-T** [comment]
**GitHubTags**:#A11yMAS;#A11yTCS;#Win32;#FTP;#DesktopApp;#A11ySev3;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:Dropdown;#CEACCApproved;#Visual Studio Code Client;#BM-VisualStudioCodeClient-Win32-Aug2022;

**Comment by roblourens** [comment]
I can no longer repro this with voiceover
